File: mycollab-mobile/src/main/scala/com/mycollab/mobile/module/project/events/TicketEvent.scala
package com.mycollab.mobile.module.project.events

import com.mycollab.events.ApplicationEvent

/**
  * @author MyColab Ltd
  * @since 5.4.3
  */
object TicketEvent {
  
  class GotoDashboard(source: AnyRef, data: AnyRef) extends ApplicationEvent(source, data) {}
  
}

File: mycollab-mobile/src/main/java/com/mycollab/mobile/module/project/events/TicketEvent.kt
package com.mycollab.mobile.module.project.events

import com.mycollab.events.ApplicationEvent
import com.mycollab.module.project.domain.criteria.ProjectTicketSearchCriteria

/**
 * @author MyCollab Ltd
 * @since 6.0.0
 */
class TicketEvent {
    class GotoDashboard(source: Any, data: ProjectTicketSearchCriteria) : ApplicationEvent(source, data) {}
}